--- query.rb_	2022-05-17 00:09:58.000000000 +0000
+++ query.rb	2022-05-19 02:39:11.488891582 +0000
@@ -1565,11 +1565,12 @@
   end
 
   def date_for_user_time_zone(y, m, d)
-    if tz = User.current.time_zone
-      tz.local y, m, d
-    else
-      Time.local y, m, d
-    end
+    # https://www.redmine.org/issues/23189
+    # if tz = User.current.time_zone
+    #   tz.local y, m, d
+    # else
+    Time.local y, m, d
+    # end
   end
 
   # Returns a SQL clause for a date or datetime field.
